lib/java/persistance/src/main/java/org/enso/persist/PerGenerator.java

```diff
@@ -235,15 +235,18 @@ private void dump(System.Logger log, int length) {
             return a.getValue()[0] - b.getValue()[0];
           });
 
-      log.log(System.Logger.Level.DEBUG, "==== Top Bytes & Counts of Classes =====");
+      log.log(System.Logger.Level.TRACE, "==== Top Bytes & Counts of Classes =====");
```

Comment:
What about wrapping the whole `dump` method in `if (log.isLoggable(Level.TRACE))`?